pyfarm/agent/etc/agent.yml

```diff
@@ -130,6 +130,13 @@ agent_http_retry_delay_factor: 5
 # be False.
 agent_http_persistent_connections: True
 
+# When using persistent connections, twisted will sometimes run requests over
+# connections that have already been closed by the server without realizing it.
+# When this happens, we catch the resulting failure and retry the request.
+# We do this up to $agent_broken_connection_max_retry times, assuming that if
```

Comment:
`$agent_broken_connection_max_retry` -> `broken_connection_max_retry`?

I'm ok with either one, the doc string just needs to match the key.  Normally I'd suggest `agent_broken_connection_max_retry` for consistency with other vars. but since this is more for internal use I don't think the prefix is required.